Update brackets

manbeneat opened the pull request:
Hi, incorrect brackets were used here so links to the targeted sections were not working :)

vgiurgiu commented:
Closing this PR, we manually copied over the content from this PR into internal GitHub and published the documentation.

manbeneat committed:
Update brackets

Hi, incorrect brackets were used here so links to the targeted sections were not working :)

vfalon commented:
Thanks for opening the PR! Closing it now, since it was fixed.
